action.yml

```diff
@@ -5,15 +5,15 @@ description: "convert golang coverage to lcov format"
 inputs:
   infile:
     description: "go coverage input file"
-    required: true
+    required: false
```

Comment:
not suggested, it is need `required: true`

but because filled default value, for user is **optional**.